### PR TITLE
Added installation steps and badges

### PR DESCRIPTION
**Pull request by mdallago**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,63 @@
-# bitprim-insight
+# Bitprim Insight <a target="_blank" href="http://semver.org">![Version][badge.version]</a> <a target="_blank" href="https://travis-ci.org/bitprim/bitprim-insight">![Travis status][badge.Travis]</a> [![Appveyor Status](https://ci.appveyor.com/api/projects/status/github/bitprim/bitprim-insight?svg=true&branch=master)](https://ci.appveyor.com/project/bitprim/bitprim-insight) <a target="_blank" href="https://gitter.im/bitprim/Lobby">![Gitter Chat][badge.Gitter]</a>
 
-REST API, matching insight API interface, implemented in .NET Core 2.x
+> Multi-Cryptocurrency _Rest_ API.
 
+*Bitprim Insight* is a REST API written in _C#_ with .NET Core 2.x which exposes methods matching the insight API interface
 
-# API HTTP Endpoints
+Bitprim Insight supports the following cryptocurrencies:
```
Comment (fpelliccioni) [inline README.md]: Does Bitprim Insight support LTC?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,63 @@
...
+## Installation Procedure
+
+The *Bitprim* libraries can be installed on Linux, macOS, FreeBSD, Windows and others. These binaries are pre-built for the most usual operating system/compiler combinations and hosted in an online repository. If there are no pre-built binaries for your platform, a build from source will be attempted.
+
+1. Build 
+
+For Bitcoin Cash
+
+```
+dotnet build /property:Platform=x64 /p:BCH=true -c Release -f netcoreapp2.0 -v normal
```
Comment (fpelliccioni) [inline README.md]: This command have to be executed in the directory where the source is, right?

File: README.md (hunk 1 above)
Comment (mdallago) [reply README.md]: Not yet...We need to release bitprim-cs for LTC first

**Comment by mdallago** [comment]
@dario-ramos Please check the english grammar

**Comment by mdallago** [comment]
@garceri Please check the build and installation procedure

File: README.md [hunk 3]
```diff
@@ -1,9 +1,201 @@
...
+dotnet publish /property:Platform=x64 /p:BTC=true -c Release -f netcoreapp2.0 -v normal -o published
+```
+
+```
+dotnet bin/x64/Release/netcoreapp2.0/published/bitprim.insight.dll --server.port=3000 --server.address=0.0.0.0
+```
+
+## Configuration Options
+
+You need to create an appsettings.json file in the build directory to run the application. You can use appsettings.example.json as starting point.
```
Comment (dario-ramos) [inline README.md]: as a starting point

File: README.md [hunk 4]
```diff
@@ -1,9 +1,201 @@
...
+        {
+          "outputTemplate" : "[{Timestamp:yyyy-MM-dd HH:mm:ss} {TimeZone}] {Level:u3} {SourceIP} {RequestId} {HttpMethod} {RequestPath} {HttpProtocol} {HttpResponseStatusCode} {HttpResponseLength} {ElapsedMs} {Message:lj}{NewLine}{Exception}"
+        }
+      }
+    ],
+    "Enrich": ["FromLogContext"]
+  }
+}
+```
+The application has two different modes of operations. As a **Full Node** or a **Forwarder**.
```
Comment (dario-ramos) [inline README.md]: two different operation modes

File: README.md [hunk 5]
```diff
@@ -1,9 +1,201 @@
...
+
+In **Full Node** mode, the application start a full bitprim node, generating a copy of the blockchain.
```
Comment (dario-ramos) [inline README.md]: the application starts a full Bitprim node

File: README.md [hunk 6]
```diff
@@ -1,9 +1,201 @@
...
+
+In **Forwarder** mode, the application only relay the request to a **Full Node** application.
```
Comment (dario-ramos) [inline README.md]: the application only relays the request

File: README.md [hunk 7]
```diff
@@ -1,9 +1,201 @@
...
+
+### Settings
+
+**ApiPrefix**: Define the name of the url segment where you expose the api methods.
```
Comment (dario-ramos) [inline README.md]: Defines the name

File: README.md [hunk 8]
```diff
@@ -1,9 +1,201 @@
...
+```
+http://blockdozer.com/[ApiPrefix]/blocks/
+```
+
+**AcceptStaleRequests**: Allow the api to respond to request although the chain is stale.
```
Comment (dario-ramos) [inline README.md]: Allows the API to respond to requests even if the chain is stale (<clarify what stale means here>)

Comment (dario-ramos) [reply README.md]: Clarify what stale means in this context

File: README.md [hunk 9]
```diff
@@ -1,9 +1,201 @@
...
+### Settings
+
+**ApiPrefix**: Define the name of the url segment where you expose the api methods.
+```
+http://blockdozer.com/[ApiPrefix]/blocks/
+```
+
+**AcceptStaleRequests**: Allow the api to respond to request although the chain is stale. 
+
+**AllowedOrigins**: Configure the allowed CORS origins.
```
Comment (dario-ramos) [inline README.md]: For multiple origins, separate them with semicolon (;)

File: README.md [hunk 10]
```diff
@@ -1,9 +1,201 @@
...
+
+**Connections**: Configure the value returned in the *connection* element of /status request.
```
Comment (dario-ramos) [inline README.md]: Configures the value [...] of the /status request

File: README.md [hunk 11]
```diff
@@ -1,9 +1,201 @@
...
+http://blockdozer.com/[ApiPrefix]/blocks/
+```
+
+**AcceptStaleRequests**: Allow the api to respond to request although the chain is stale. 
+
+**AllowedOrigins**: Configure the allowed CORS origins.
+
+**Connections**: Configure the value returned in the *connection* element of /status request. 
+
+**DateInputFormat**: Define the date format used by /blocks and others requests that requieres dates.
```
Comment (dario-ramos) [inline README.md]: Defines the date format [...] and other requests that require dates

File: README.md [hunk 12]
```diff
@@ -1,9 +1,201 @@
...
+
+**EstimateFeeDefault**: Set the value returned by /utils/estimatefee.
```
Comment (dario-ramos) [inline README.md]: Sets the value

File: README.md [hunk 13]
```diff
@@ -1,9 +1,201 @@
...
+
+**ForwardUrl**: When you use the application in **Forwarder** mode. This setting set the url of a **Full Node**.
```
Comment (dario-ramos) [inline README.md]: [...] mode, this settings defines the Full Node's URL

File: README.md [hunk 14]
```diff
@@ -1,9 +1,201 @@
...
+
+**Connections**: Configure the value returned in the *connection* element of /status request. 
+
+**DateInputFormat**: Define the date format used by /blocks and others requests that requieres dates.
+
+**EstimateFeeDefault**: Set the value returned by /utils/estimatefee.
+
+**ForwardUrl**: When you use the application in **Forwarder** mode. This setting set the url of a **Full Node**. 
+
+**InitializeNode**: This setting define the working mode of the node. *True* for Full Node or *False* for Forwarder Node.
```
Comment (dario-ramos) [inline README.md]: This setting defines the node's working mode: True for full node, [...]

File: README.md [hunk 15]
```diff
@@ -1,9 +1,201 @@
...
+
+**LongResponseCacheDurationInSeconds**: Duration of the long cache responses. Used by: 
+* /rawblock 
+* /rawtx
+ 
+
+**MaxBlockSummarySize**: Define the max limit of the /blocks method.
```
Comment (dario-ramos) [inline README.md]: Defines the max

File: README.md [hunk 16]
```diff
@@ -1,9 +1,201 @@
...
+
+**MaxCacheSize**: Configure the size limit of the cache.
+
+**NodeConfigFile**: The path of the config file used by the node. Only use in **Full Node** mode.
```
Comment (dario-ramos) [inline README.md]: Node config file path; can be absolute, or relative to the project directory

File: README.md [hunk 17]
```diff
@@ -1,9 +1,201 @@
...
+
+**NodeType**: The value returned in *type* element by /sync method.
+
+**PoolsFile**: Path to the json file with the mining pool information.
+
+**ProtocolVersion**: The value returned in *protocolversion* element by /status method.
```
Comment (dario-ramos) [inline README.md]: the /status method

File: README.md [hunk 18]
```diff
@@ -1,9 +1,201 @@
...
+
+**Proxy**: The value returned in *proxy* element by /status method.
```
Comment (dario-ramos) [inline README.md]: the /status method

File: README.md [hunk 19]
```diff
@@ -1,9 +1,201 @@
...
+
+**RelayFee**: The value returned in *relayfee* element by /status method.
```
Comment (dario-ramos) [inline README.md]: the /status method

File: README.md [hunk 20]
```diff
@@ -1,9 +1,201 @@
...
+* /addr/{paymentAddress}/totalSent
+* /addr/{paymentAddress}/unconfirmedBalance
+* /addr/{paymentAddress}/utxo
+* /addrs/{paymentAddresses}/utxo
+* /addrs/utxo
+* /addr/{paymentAddress}
+* /peer
+* /version
+
+**TimeOffset**: The value returned in *timeoffset* element by /status method.
```
Comment (dario-ramos) [inline README.md]: the /status method

File: README.md [hunk 21]
```diff
@@ -1,9 +1,201 @@
...
+
+**TransactionsByAddressPageSize**: The max page limit used by /txs method.
```
Comment (dario-ramos) [inline README.md]: the /txs method

File: README.md [hunk 22]
```diff
@@ -1,9 +1,201 @@
...
+* /addrs/utxo
+* /addr/{paymentAddress}
+* /peer
+* /version
+
+**TimeOffset**: The value returned in *timeoffset* element by /status method.
+
+**TransactionsByAddressPageSize**: The max page limit used by /txs method. 
+
+**Version**: The value returned in *version* element by /status method.
```
Comment (dario-ramos) [inline README.md]: the /status method

File: README.md [hunk 23]
```diff
@@ -1,9 +1,201 @@
...
+* /peer
+* /version
+
+**TimeOffset**: The value returned in *timeoffset* element by /status method.
+
+**TransactionsByAddressPageSize**: The max page limit used by /txs method. 
+
+**Version**: The value returned in *version* element by /status method. 
+
+**HttpClientTimeoutInSeconds**: Define HttpClient timeout. Used by the forwarders.
```
Comment (dario-ramos) [inline README.md]: Defines

File: README.md [hunk 24]
```diff
@@ -1,9 +1,201 @@
...
+
+**TimeOffset**: The value returned in *timeoffset* element by /status method.
+
+**TransactionsByAddressPageSize**: The max page limit used by /txs method. 
+
+**Version**: The value returned in *version* element by /status method. 
+
+**HttpClientTimeoutInSeconds**: Define HttpClient timeout. Used by the forwarders. 
+
+**Serilog**: The serilog configuration. For more documentation check https://github.com/serilog/serilog/wiki/Getting-Started
```
Comment (dario-ramos) [inline README.md]: The Serilog configuration. For more detailed documentation, check [...]

File: README.md (hunk 2 above)
Comment (garceri) [reply README.md]: Should read 
`The *Bitprim* libraries can be installed using conan (see below) on Linux, macOS, FreeBSD, Windows and others. These binaries are pre-built for the most usual operating system/compiler combinations and are downloaded from an  online repository. If there are no pre-built binaries for your platform, conan will attempt to build from source during the installation`

**Pull request merged**